core/matching/side.go

```diff
@@ -133,6 +133,33 @@ func (s *OrderBookSide) BestStaticPriceAndVolume() (*num.Uint, uint64, error) {
 	return num.UintZero(), 0, errors.New("no non pegged orders found on the book")
 }
 
+func (s *OrderBookSide) amendIcebergOrder(amendOrder *types.Order, oldOrder *types.Order, priceLevelIndex int, orderIndex int) (uint64, error) {
+	if amendOrder.Remaining > oldOrder.Remaining {
+		// iceberg amend should never increase the visiable remaining
```

Comment:
how come the linter is not complaining about visiable?

Reply:
looks like I spelt it _too_ wrong: https://github.com/client9/misspell/blob/master/words.go#L27308